docs/en/guides/contribution/how-to-release.md

```diff
@@ -0,0 +1,182 @@
+# Apache SkyWalking Rover Release Guide
+
+This documentation guides the release manager to release the SkyWalking Rover in the Apache Way, and also helps people to check the release for vote.
+
+## Prerequisites
+
+1. Close(if finished, or move to next milestone otherwise) all issues in the current milestone from [skywalking-rover](https://github.com/apache/skywalking-rover/milestones) and [skywalking](https://github.com/apache/skywalking/milestones), create a new milestone if needed.
+2. Update [CHANGES.md](../../../../CHANGES.md).
```

Comment:
Please add dependencies check as step (3).

Reply:
Do you mean dependency license check? been added.